Anastasiya Santalova/grid/SCSS/hw2_grid.scss

```diff
@@ -0,0 +1,316 @@
+@import "variables";
+@import "mixin";
+
+
+// CSS Styles
+
+html {
+  @include fullScreen;
+}
+
+body {
+  box-sizing: border-box;
+
+  @include fullScreen;
+
+  min-width: 320px;
+  min-height: 320px;
+
+  margin: 0;
+  padding: 30px;
+
+  @include font(15pt, 600);
+
+  background-image: url(img/main-background.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: center center;
+}
+
+.widget {
+  @include fullScreen;
+  
+  @include gridThreeColumns();
+}
+
+.weather {
+  @include fullScreen;
+
+  display: flex;
+  align-content: center;
+  align-items: center;
+}
+
+.weather-skyblue {
+  grid-area: wskyblue;
+
+  background-color: $skyBlue;
+}
+
+.weather-green {
+  grid-area: wgreen;
+
+  background-color: $green;
+}
+
+.weather-blue {
+  grid-area: wblue;
+
+  background-color: $blue;
+}
+
+.weather-orange {
+  grid-area: worange;
+
+  background-color: $orange;
+}
+
+.weather-pink {
+  grid-area: wpink;
+  
+  background-color: $pink;
+}
+
+.weather-purple {
+  grid-area: wpurple;
+
+  background-color: $purple;
+}
+
+// Visible elements before hover
+
+.weather__block_visible {
```

Comment:
I agree with your comment, but didn't fix so much, because my current structure of nesting does not allow to improve it without changing all code.

Reply:
Tell me please, how can I check my markup on IE11, if I work on Mac?

Reply:
Actually I didn't work on Mac so I can't advice you anything specific. Take a look at this article:
https://dzone.com/articles/7-ways-you-can-test-your-website-on-internet-explo

Maybe it will help, the first point looks quite simple and well-working at the same time

Reply:
Also I don't insist on renaming all the classname using SASS features, it still looks good, I just gave you the reommendation for future :)